hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/statemachine/commandhandler/CloseContainerCommandHandler.java

```diff
@@ -102,7 +102,12 @@ public void handle(SCMCommand command, OzoneContainer ozoneContainer,
         final Container container = controller.getContainer(containerId);
 
         if (container == null) {
-          LOG.error("Container #{} does not exist in datanode. "
+          // Change the log level from error to info,
+          // since the container may not have been created
+          // due to issue documented in HDDS-9550/HDDS-9600.
+          // Currently the error log stands out when it is
+          // probably not a real issue.
+          LOG.info("Container #{} does not exist in datanode. "
               + "Container close failed.", containerId);
```

Comment:
Let's try to keep the log message self documenting so it's easier for people looking at logs without the code.
```suggestion
          LOG.info("Container #{} does not exist in datanode. Its pipeline may have closed before it was created."
              + "Container close failed.", containerId);
```

Reply:
sounds good! thanks @errose28 , updated.